tests/test_product.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from buy_together_app.models import Product
+
+QR = "Q5o76MbdiNbXprNEnHfpcGWFp1CMF8XY"
+NAME = "Banana"
+DESCRIPTION = "Good!"
+
+
+@pytest.fixture
+def product0():
+    return Product(qr_code=QR, product_name=NAME, description=DESCRIPTION)
+
+
+@pytest.fixture
+def saved_product0(product0):
+    product0.save()
+    return product0
+
+
+test_data = [
+            Product('5yMcE3UrEEmZBukWZBrlCKnu0K5j24UY', 'Apple', 'Sweety!'),
+            Product('Q5o76MbdiNbXprNEnHfpcGWFp1CMF8ad', 'Banana', "Good!"),
+            Product(qr_code=QR, product_name=NAME, description=DESCRIPTION),
+        ]
+
+
+class TestProductModel:
+    @pytest.mark.django_db()
+    def test_save_product(self, saved_product0):
+        assert saved_product0 in Product.objects.all()
+
+    @pytest.mark.django_db()
+    def test_delete_product(self, saved_product0):
+        assert saved_product0 in Product.objects.all()
+        saved_product0.delete()
+        assert saved_product0 not in Product.objects.all()
+
+    @pytest.mark.django_db()
+    def test_none_feilds(self, saved_product0):
```

Comment:
typo: `feilds` -> `fields`

Reply:
What's the purpose of this test?

Reply:
Deleted and done it better in test validations